ts/tool-circularize.ts

```
@@ -1,7 +1,6 @@
 import {angleBetweenVectors, distance, normalizeVector} from './Vector';
-import {cubicBezierMinRadius, hermiteToBezier} from './util-bezier';
-import {HermiteCurve} from './util-bezier';
-import {fp32v} from './util';
+import {HermiteCurve, cubicBezierMinRadius, hermiteToBezier} from './util-bezier';
```

ESLint

[error] Imports should be sorted alphabetically.

(sort-imports)